Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM bitnami/python:3.8.8
```

Comment:
The python version can be updated to 3.9.13. It is working without giving any warnings for me. Is it fine with you? @StefanoStone